packages/x-components/tests/e2e/cucumber/partial-results/partial-results.feature

```diff
@@ -1,7 +1,11 @@
 Feature: Partial results component
 
+  Background:
```

Comment:
I think in this case we can remove the `Background` as we're doing more requests than we should and move this `Given` step to the Scenarios 2 and 3 just before the `And  no special config for partial-results view`.